write-fonts/src/tables/glyf.rs

```diff
@@ -295,4 +344,81 @@ mod tests {
         assert_eq!(points[4].y, 256);
         assert!(points[4].on_curve);
     }
+
+    #[test]
+    fn compile_repeatable_flags() {
+        let mut path = BezPath::new();
+        path.move_to((20., -100.));
+        path.line_to((80., -20.));
+        path.line_to((50., -69.));
+        path.line_to((25., -90.));
+        // before reversal/rotation: A B C D
+        // after reversal D C B A
+        // after r-rot-1 A D C B
+
+        let glyph = SimpleGlyf::from_kurbo(&path).unwrap();
+        let flags = glyph
+            .compute_point_deltas()
+            .map(|x| x.0)
+            .collect::<Vec<_>>();
+        let r_flags = RepeatableFlag::iter_from_flags(flags.iter().copied()).collect::<Vec<_>>();
+
+        assert_eq!(r_flags.len(), 2, "{r_flags:?}");
+        let bytes = crate::dump_table(&glyph).unwrap();
+        let read = read_fonts::tables::glyf::SimpleGlyph::read(FontData::new(&bytes)).unwrap();
+        assert_eq!(read.number_of_contours(), 1);
+        assert_eq!(read.num_points(), 4);
+        assert_eq!(read.end_pts_of_contours(), &[3]);
+        let points = read.points().collect::<Vec<_>>();
+        assert_eq!(points[0].x, 20);
+        assert_eq!(points[0].y, -100);
+        assert_eq!(points[1].x, 25);
+        assert_eq!(points[1].y, -90);
+        assert_eq!(points[2].x, 50);
+        assert_eq!(points[2].y, -69);
+        assert_eq!(points[3].x, 80);
+        assert_eq!(points[3].y, -20);
```

Comment:
I'm curious why we don't assert `read.points().collect::<Vec<_>>()` equals `vec![Point(..), etc]`?